libs/novui/src/json-schema-components/JsonSchemaForm.tsx

```diff
@@ -1,68 +1,72 @@
-import React from 'react';
-import { RJSFSchema } from '@rjsf/utils';
+import Form, { FormProps } from '@rjsf/core';
```

Comment:
I'd argue that the JSONSchemaForm should be part of our UI library, as it is very specific to the studio part of the web app but we can debate on that later.

Reply:
Sounds good!